datafusion/core/tests/window.rs

```diff
@@ -0,0 +1,140 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I wonder if we can make this a submodule under core_integration (each test binary takes up non trivial disk space and a while ago we hit runner disk space limits)

Aka make it a module in a subdirectory as part of 
https://github.com/apache/arrow-datafusion/blob/main/datafusion/core/tests